### PR TITLE
Update README_cn.md

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**

### CODE REVIEW

**Comment by OpenVINO-dev-contest** [comment]
@zhuo-yoyowz Please help to review, thanks

File: README_cn.md [hunk 1]
```diff
@@ -23,8 +23,9 @@
 | [DeepFloyd IF - Text-to-Image generation](notebooks/238-deeployd-if/)<br>| 利用DeepFloyd IF以及OpenVINO™进行文本到图像生成 | <img src=https://user-images.githubusercontent.com/29454499/241643886-dfcf3c48-8d50-4730-ae28-a21595d9504f.png width=300> |   |
 | [ImageBind](notebooks/239-image-bind/)<br>| 使用ImageBind以及OpenVINO™结合多模态数据 | <img src=https://user-images.githubusercontent.com/29454499/240364108-39868933-d221-41e6-9b2e-dac1b14ef32f.png width=300> |   |
 | [Dolly v2](notebooks/240-dolly-2-instruction-following/)<br>| 使用Databricks Dolly 2.0以及OpenVINO™遵循指令进行文本生成 | <img src=https://user-images.githubusercontent.com/29454499/237291423-022f07d2-966b-4be2-9a1c-98f1cf0691c2.png width=300> | |
-
-[![-----------------------------------------------------](https://user-images.githubusercontent.com/10940214/155750931-fc094349-b6ec-4e1f-9f9a-113e67941119.jpg)]()
+| [Stable Diffusion XL](notebooks/248-stable-diffusion-xl/)<br>| 使用Stable Diffusion X以及OpenVINO™实现图像生成 | <img src=https://user-images.githubusercontent.com/29454499/258651862-28b63016-c5ff-4263-9da8-73ca31100165.jpeg width=300> | |
```
Comment (zhuo-yoyowz) [inline README_cn.md:26]:
```suggestion
| [Stable Diffusion XL](notebooks/248-stable-diffusion-xl/)<br>| 使用Stable Diffusion XL以及OpenVINO™实现图像生成 | <img src=https://user-images.githubusercontent.com/29454499/258651862-28b63016-c5ff-4263-9da8-73ca31100165.jpeg width=300> | |
```

File: README_cn.md [hunk 2]
```diff
@@ -155,6 +156,15 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [244-named-entity-recognition](notebooks/244-named-entity-recognition/)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/244-named-entity-recognition/244-named-entity-recognition.ipynb) | 使用OpenVINO™进行命名实体识别 | |
+| [245-typo-detector](notebooks/245-typo-detector/)<br>| 使用OpenVINO™进行英文文本纠错 | <img src=https://user-images.githubusercontent.com/80534358/224564463-ee686386-f846-4b2b-91af-7163586014b7.png   width=225> |
+| [246-depth-estimation-videpth](notebooks/246-depth-estimation-videpth/)<br>| 基使用OpenVINO™进行基于视觉的单目深度估测 | <img src=https://raw.githubusercontent.com/alexklwong/void-dataset/master/figures/void_samples.png width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:162]:
```suggestion
| [246-depth-estimation-videpth](notebooks/246-depth-estimation-videpth/)<br>| 使用OpenVINO™进行基于视觉的单目深度估测 | <img src=https://raw.githubusercontent.com/alexklwong/void-dataset/master/figures/void_samples.png width=225> |
```

**Comment by eaidova** [comment]
Hi @OpenVINO-dev-contest @zhuo-yoyowz, thank you for update, is it ready for merge?

**Comment by OpenVINO-dev-contest** [comment]
yes I think so

**Review by eaidova — APPROVED** [review]

**Pull request merged**